index.js

```diff
@@ -3,27 +3,39 @@ const github = require('@actions/github');
 const AWS = require('aws-sdk')
 const fs = require('fs')
 
-const dynamodb = new AWS.DynamoDB.DocumentClient({apiVersion: '2012-08-10'});
-const {execSync} = require("child_process");
+const dynamodb = new AWS.DynamoDB.DocumentClient({ apiVersion: '2012-08-10' });
+const { execSync } = require("child_process");
 
 const gitLatestTagCommand = 'git tag --list --sort=-version:refname "v*" --merged | head -n 1';
-
 (async () => {
     try {
         const stack = core.getInput('stack');
         const reportId = core.getInput('report-id');
         const tableName = core.getInput('table-name');
 
-        let versionDetails = await processVersions(tableName, reportId, stack)
+        let branchName = getBranchName()
+
+        //ReGex for dev/** or tools/**
+        let re = /^(dev|tools)\/.*/
 
-        core.setOutput("version-details", versionDetails);
-        const payload = JSON.stringify(github.context.payload, undefined, 2)
+        let rushFile = fs.readFileSync(`rush.json`)
+        let rush = JSON.parse(rushFile)
+        let repositoryName = rush["repository"]["url"].split('/').pop()
+        delete rushFile
+
+        if (repositoryName == "mosaic-shell-modules" && re.test(branchName)) {
```

Comment:
Could we change it to an optional input variable that would specify that this is a "multi-branch report" or something similar? I would prefer not hardcoding a specific repo in the code here.

Reply:
updated with the latest code